opennms-base-assembly/pom.xml

```diff
@@ -164,6 +164,10 @@
                 <groupId>eclipse</groupId>
                 <artifactId>jdtcore</artifactId>
               </exclusion>
+              <exclusion>
+                <groupId>com.lowagie</groupId>
+                <artifactId>itext</artifactId>
+              </exclusion>
```

Comment:
That seems subtle. Where did this come from?

Reply:
I encountered issues getting this library (transitive dependency of jasper reports) due to `maven-default-http-blocker` blocking http requests. I believe this forces Maven to use a different repository to get this artifact, i.e. one of the https repositories. However could just be due to issues we were having with the OpenNMS repository. We could see if this gets resolved and remove this entry eventually.

Reply:
We should definitely check, in order not to break something somewhere else.

Reply:
I will remove this and see if things build on CI server.

Reply:
there was an outage on the upstream maven repository, it _should_ be OK now, but if you still have problems let me know and I'll dig into it more

Reply:
Thanks will do!

Reply:
Removed this